[2025-03-23T22:16:13.126+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-23T22:16:13.150+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2021-11-28T00:00:00+00:00 [queued]>
[2025-03-23T22:16:13.158+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2021-11-28T00:00:00+00:00 [queued]>
[2025-03-23T22:16:13.159+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-23T22:16:13.172+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2021-11-28 00:00:00+00:00
[2025-03-23T22:16:13.177+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1511) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-23T22:16:13.178+0000] {standard_task_runner.py:72} INFO - Started process 1526 to run task
[2025-03-23T22:16:13.180+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v02', 'get_name', 'scheduled__2021-11-28T00:00:00+00:00', '--job-id', '161', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmpedj09baw']
[2025-03-23T22:16:13.182+0000] {standard_task_runner.py:105} INFO - Job 161: Subtask get_name
[2025-03-23T22:16:13.229+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2021-11-28T00:00:00+00:00 [running]> on host 4f1e28758fdd
[2025-03-23T22:16:13.287+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v02' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2021-11-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-11-28T00:00:00+00:00'
[2025-03-23T22:16:13.288+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-23T22:16:13.290+0000] {python.py:240} INFO - Done. Returned value was: {'first_name': 'Samuel', 'last_name': 'Ferreira'}
[2025-03-23T22:16:13.314+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-23T22:16:13.314+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v02, task_id=get_name, run_id=scheduled__2021-11-28T00:00:00+00:00, execution_date=20211128T000000, start_date=20250323T221613, end_date=20250323T221613
[2025-03-23T22:16:13.359+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-23T22:16:13.371+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-23T22:16:13.372+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
